dist/debuginfo/scripts/create-relocatable-package.py

```diff
@@ -56,7 +56,10 @@ def reloc_add(ar, name, arcname=None):
 
 ar = tarfile.open(fileobj=gzip_process.stdin, mode='w|')
 # relocatable package format version = 2.1
-shutil.rmtree(f'build/{SCYLLA_DIR}', ignore_errors=True)
+try:
+    shutil.rmtree(f'build/{SCYLLA_DIR}')
+except FileNotFoundError:
+    pass
 os.makedirs(f'build/{SCYLLA_DIR}')
```

Comment:
Will this not just break the build?

Reply:
how so? we always `rmtree/{SCYLLA_DIR}` and we bail out if it fails.

Reply:
Yes. I guess the failure will be due to permission errors, and the permission errors will be due to unpacking tarballs without dropping their uid assignments.

Reply:
Furthermore, ignore_errors is usually a bad idea.

Reply:
> Yes. I guess the failure will be due to permission errors, and the permission errors will be due to unpacking tarballs without dropping their uid assignments.

yeah, your fix has been merged. very likely. anyway, fail early should helps us identify issues in this area if more FTBFS surfaces.